plugins/check_snmp.c

```diff
@@ -1186,16 +1190,17 @@ multiply (char *str)
 	if (fmtstr != "") {
 		conv = fmtstr;
 	}
+	buf = malloc(100);
```

Comment:
How about we just have this as a static buffer so we do not deal with the malloc? We would have 100 byte leaks for every line in snmpgets output.

Reply:
sounds good, will do